libminifi/include/processors/GetFile.h

```diff
@@ -194,7 +194,7 @@ class GetFile : public core::Processor, public state::response::MetricsNodeSourc
   std::shared_ptr<logging::Logger> logger_;
 };
 
-REGISTER_RESOURCE(GetFile);
+REGISTER_RESOURCE(GetFile,"Creates FlowFiles from files in a directory. NiFi will ignore files for which it doesn't have read permissions.");
```

Comment:
MiNiFi?

Reply:
Yah. These are auto copied from processors.md, so these typos exist there too. Thank for catching.

Reply:
ha actually this one I modified as it ended in a prepositional phrase but I didn't notice the NiFi. Thanks!